README.md

```diff
@@ -65,6 +65,29 @@ The app automatically launches with the Chromium developer tools opened by defau
 
 Code formatting has been set up to make merging PRs easier. It uses the same prettier/eslint mechanism as Calypso. See [JavaScript Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md) for details on setting up your editor.
 
+### Testing
+
+#### Running tests
+
+You can run tests with the following command:
+
+```bash
+npm run test
+```
+
+There are also E2E tests available. They require a little bit more setup - clean the `out/` directory and build the app binary: 
+
+```bash
+rm -rf out/
+npm run make
+```
+
+And then run tests:
+
+```bash
+npm run e2e
```

Comment:
We could mention that with the command `npm run e2e -- <test-filter>` (e.g. `npm run e2e -- app.test`) specific tests can be run. This would be useful when working on creating/updating e2e tests.

Reply:
It makes sense, but let's leave it as is for now to avoid making the readme too crowded. If the developer starts working on code and wants to either use watch mode, or run a single test, they will figure it out. The main reason of adding this section is to mark that there are tests available and to invite developer to dig more.